packages/core/test/unit/aws-xray.test.js

```diff
@@ -44,7 +44,17 @@ describe('AWSXRay', function() {
 
       assert.property(setSDKDataStub.firstCall.args[0], 'sdk');
       assert.property(setSDKDataStub.firstCall.args[0], 'sdk_version');
+      assert.notStrictEqual(
+        setSDKDataStub.firstCall.args[0].sdk_version,
+        'unknown',
+        'Expected sdk_version to not be unknown'
+      );
       assert.property(setSDKDataStub.firstCall.args[0], 'package');
+      assert.notStrictEqual(
+        setSDKDataStub.firstCall.args[0].package,
+        'unknown',
+        'Expected package to not be unknown'
+      );
```

Comment:
@willarmiros I'm not super familiar with chai and didn't spend that long googling. I know in Jest I could do something that would assert the structure of this object and the values of it's key's in one shot, I'm not sure if there's an equivalent in chai or if this approach of independent asserts is prefered.

Reply:
You can probably use [hasAllKeys](https://www.chaijs.com/api/assert/#method_hasallkeys) instead of asserting for individual properties how we're doing it now, but as for checking that they're not unknown the way you're doing it now seems fine.

Reply:
going to leave this as-is then.